src/lib/utils.js

```diff
@@ -533,14 +534,14 @@ function addGravatarSupport(pkgInfo: Object): Object {
  * @return {String} converted html template
  */
 function parseReadme(packageName: string, readme: string): string {
-  if (readme) {
-    return marked(readme);
+  if (_.isEmpty(readme) === false) {
```

Comment:
I am not a big fan of complete `lodash` import and `lodash` in general ;-)

Reply:
🤔 I'm lost.

Reply:
In v3 I'm not that strict anyway, we use `lodash` all over the place :-)